Fullstory v2 docs (#5056)
segmentio/segment-docs: src/connections/destinations/catalog/actions-fullstory-cloud/index.md

### Getting Started

> 1. You need a FullStory API Key to use the FullStory cloud mode destination. Refer to [this article](https://help.fullstory.com/hc/en-us/articles/360052021773-Managing-API-Keys){:target="_blank"} to learn how to generate a new API Key within FullStory. You need an `Admin` or `Architect` API Key for GDPR deletions to propagate. You may also contact FullStory directly for deletions.

Reviewer: The GDPR deletions information seems important - let's pull it out and add it to the "Troubleshooting" section for visibility!

Is there a minimum level of access you would need? For example, would you need a key with architect or admin permissions to link to the destination within Segment?

```suggestion
1. You need a FullStory API Key to use the FullStory cloud mode destination. Refer to [this article](https://help.fullstory.com/hc/en-us/articles/360052021773-Managing-API-Keys){:target="_blank"} to learn how to generate a new API Key within FullStory.
```

### Why am I getting a ‘404 Not Found’ error?

> This applies to the orginal 'Identify User' and 'Track Event' actions. The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.

Reviewer: ```suggestion
If you are using the original 'Identify User' and 'Track Event' actions and encounter a "404 Not Found" error, the user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
```

Data sent server-side for users must match an already existing userId that was sent from a client-side connection.

The new 'Identify User V2' and 'Track Event V2' actions will automatically create users when a user matching the
provided UID is not found.

> 

Reviewer: ```suggestion

### Why can't I propagate GDPR deletions?

GDPR deletions require an `Admin` or `API` key to propagate. You may also contact FullStory directly for deletions. 

```
